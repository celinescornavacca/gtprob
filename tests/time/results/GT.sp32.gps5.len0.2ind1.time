2.49user 0.16system 0:02.89elapsed 91%CPU (0avgtext+0avgdata 53196maxresident)k
138inputs+0outputs (0major+26290minor)pagefaults 0swaps
