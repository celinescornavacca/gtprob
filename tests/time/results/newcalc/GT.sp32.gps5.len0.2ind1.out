-295.282389683
